lib/dynamic_config/datastore_cache.rb

```diff
@@ -21,6 +21,8 @@
 class DatastoreCache
   SHARED_CACHE_NAMESPACE = "DynamicConfigData".freeze
 
+  attr_reader :datastore
```

Comment:
Directly exposing the data that this class is intended to wrap seems like an anti-pattern; can this be removed? It's also not clear to me whether or not it's being used.

Reply:
ooh. Good catch. I think I added this to debug some issues and didn't put in a TODO comment about removing it.